http-client/src/main/java/io/airlift/http/client/jetty/HttpClientLogger.java

```diff
@@ -25,6 +25,7 @@
 {
     void log(RequestInfo requestInfo, ResponseInfo responseInfo);
     void close();
+    int getLoggerQueueSize();
```

Comment:
Name this `getQueueSize()`? The "logger" part seems redundant since this is the logger interface.